src/web/js/output-ui.js

```diff
@@ -1275,10 +1275,11 @@
           var originalImageDom = img.toDomNode();
           $(container).click(function(e) {
             var dialog = $("<div>");
+            // NOTE(Oak): some magic numbers that "display" nicely
```

Comment:
Ugh, those are seriously magic numbers.  Where do they come from?  Do they work on all reasonable screen resolutions and sizes?  (Especially projected screens, and laptops...)

Reply:
It's the width / height of, say, title bar of jQuery dialog. You have a good point, since I have not tested on other browser besides Firefox and Chrome, but it looks OK on those two.

Reply:
At least leave a comment explaining where you guesstimated the numbers from, otherwise we'll never be able to reconstruct this logic.  Why are you specifying the numbers here anyway, rather than nesting a div one layer deeper, and specifying the actual, correct numbers on that inner div, then let the outer one take care of itself?